superset-frontend/src/common/components/Modal/Modal.tsx

```diff
@@ -42,12 +42,15 @@ interface ModalProps {
   footer?: React.ReactNode;
   wrapProps?: object;
   height?: string;
+  scrollTableMode?: boolean;
 }
 
 interface StyledModalProps extends SupersetThemeProps {
   maxWidth?: string;
   responsive?: boolean;
   height?: string;
+  scrollTableMode?: boolean;
```

Comment:
Do we really need both props for the modal? I think it's OK to keep some bottom padding when only table body is scrollable.

Reply:
You would see the white padding at the bottom overlaying the table and creating a disturbing effect for the eyes. I am happy to change the name or pass it down differently if you have a better idea though.

Reply:
You could add a `margin-bottom: - theme.gridUnit * 4` to the scrollable table.